code/game/gamemodes/changeling/powers/lesserform.dm

```diff
@@ -21,6 +21,8 @@
 		to_chat(H, "<span class='warning'>We cannot perform this ability in this form!</span>")
 		return
 
+	var/datum/dna/DNA = changeling.GetDNA(user.dna.real_name)
+	user.dna = DNA ? DNA.Clone() : user.dna
```

Comment:
Yeah. I don't know why I thought it'd be a good idea to crossreference it with stored genomes, this might only cause problems. I'll change it now